docs/source/getting_started/index.md

```diff
@@ -56,6 +56,21 @@ docker run -it \
   --env INFERENCE_MODEL=$INFERENCE_MODEL \
   --env OLLAMA_URL=http://host.docker.internal:11434
 ```
+
+As another example, to start the container with Podman, you need to first create the `~/.llama` directory and then start Podman as follows (note that the OLLAMA_URL is slightly different too):
+
+```bash
+mkdir ~/.llama
```

Comment:
This will fail if executed the second time. Do `mkdir -p ~/.llama`. Also, I agree we should just do it for default `docker` case and just suggest to replace `docker` with `podman` as needed.

Reply:
Agree with @booxter on both points.

Reply:
Moving to `mkdir -p ~/.llama` makes sense.  I am less convinced that telling people to do it with Docker too is a good idea, but it shouldn't be a problem -- everything will still work.  I will make these updates now.

Reply:
I have made these updates now.